pkg/actions/iam/tasks.go

```diff
@@ -0,0 +1,62 @@
+package iam
+
+import (
+	"fmt"
+
+	iamoidc "github.com/weaveworks/eksctl/pkg/iam/oidc"
+
+	"github.com/weaveworks/eksctl/pkg/cfn/builder"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/cfn/manager"
+	"github.com/weaveworks/eksctl/pkg/utils/tasks"
+)
+
+func NewUpdateIAMServiceAccountTask(clusterName string, sa *api.ClusterIAMServiceAccount, stackManager StackManager, iamServiceAccount *api.ClusterIAMServiceAccount, oidcManager *iamoidc.OpenIDConnectManager) (*tasks.TaskTree, error) {
```

Comment:
`sa *api.ClusterIAMServiceAccount` and `iamServiceAccount *api.ClusterIAMServiceAccount` are duplicate arguments?

Reply:
good spot! I will fix as part of https://github.com/weaveworks/eksctl/pull/3135